test/common/index.js

```diff
@@ -802,3 +804,18 @@ exports.hijackStdout = hijackStdWritable.bind(null, 'stdout');
 exports.hijackStderr = hijackStdWritable.bind(null, 'stderr');
 exports.restoreStdout = restoreWritable.bind(null, 'stdout');
 exports.restoreStderr = restoreWritable.bind(null, 'stderr');
+
+exports.getTTYfd = function getTTYfd() {
+  const tty = require('tty');
+  // Do our best to grab a tty fd.
+  const ttyFd = [1, 2, 4, 5].find(tty.isatty);
+  if (ttyFd === undefined) {
+    try {
+      return openSync('/dev/tty');
+    } catch (e) {
+      // There aren't any tty fd's available to use.
+      return -1;
+    }
+  }
+  return ttyFd;
```

Comment:
Something similar wa sin here at some point and is used in another test file. It would be good to consolidate in that case.

Reply:
If there is, I'm not sure where it is at. If there are tests somewhere that I missed that use something like this, they probably don't actually work...? I couldn't find any in a quick look.

There are some tests that don't even bother to go for something like this and just go for fd 0 instead. (Which could probably be added to the lookup list here? I'll do that.)

Reply:
There is one in `sequential/test-async-wrap-getasyncid.js`. And zero was removed because it can cause issues on Windows.

Reply:
That part of that test definitely never ever runs. 😬

Reply:
I'll move that out next and base it off of this then, I suppose.